### DISCUSSION TITLE
Automations

### DISCUSSION THREAD

**Discussion by bkonia** [stacks-task-manager/stacks2#35] [Ideas & Feature requests]
We need some additional automations:

- Task is after start date and before due date

- Task is before start date

Also, we should be able to specify multiple conditions using and/or logic.

**Comment by bduplanty** [comment]
Would also like the ability to save an automation to apply to other projects -- have to keep creating the same automation unless there's a way to to it from copying files?

**Reply by synapse** [reply]
Hey @bduplanty this is coming soon 😉
